akka-management/core/src/main/scala/com/lightbend/lagom/internal/akka/management/AkkaManagementTrigger.scala

```diff
@@ -26,17 +27,22 @@ private[lagom] class AkkaManagementTrigger(
   coordinatedShutdown: CoordinatedShutdown
 )(implicit executionContext: ExecutionContext) {
 
-  private val enabled = config.getBoolean("lagom.akka.management.enabled")
+  private val enabledSetting = "lagom.akka.management.enabled"
+  private val enabledRenderedValue = config.getValue(enabledSetting).render()
+  private val enabled = config.getBoolean(enabledSetting)
+
+  private val logger = Logger(this.getClass)
 
   /**
    * Starts Akka HTTP Management honoring the `lagom.akka.management.enabled` setting
    */
   private[lagom] def start() = {
 
     if (enabled) {
-      forcedStart()
+      doStart()
     } else {
-      // TODO: trace a WARN to state nothing happened
+      logger.debug(
+        s"'lagom.akka.management.enabled' property is set to '$enabledRenderedValue'. Akka Management won't start.")
```

Comment:
TODO comment said: "trace a WARN", but I don't think it should be a warning. 

By default, the value is `on`, this logging appears when the user explicitly set it to `off`. Since it's an opt-out, this should only be printed when debugging. We should not warn users after they explicitly decided to disable it.